robocop/checkers/comments.py

```diff
@@ -50,19 +50,46 @@
         
         Example::
         
-        
             # good
              # bad
               # third cell so it's good
         
         """,
     ),
-    "0704": Rule(rule_id="0704", name="ignored-data", msg="Ignored data found in file", severity=RuleSeverity.WARNING),
+    "0704": Rule(
+        rule_id="0704",
+        name="ignored-data",
+        msg="Ignored data found in file",
+        severity=RuleSeverity.WARNING,
+        docs="""
+        All lines before first test data section 
+        (`ref <https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#test-data-sections>`_) 
+        are ignored. It's recommended to add `*** Comments ***` section header for lines that should be ignored.
+        
+        Missing section header::
+    
+            Resource   file.resource  # it looks like *** Settings *** but section header is missing - line is ignored
+            
+            *** Keywords ***
+            Keyword Name
+               No Operation
+        
+        Comment lines that should be inside `*** Comments ***`::
```

Comment:
It shows that you can add any free text comment before first test data section and it will be not parsed - though it's recommeneded to use `*** Comments ***` section to make it more explicit

Reply:
What is the difference between those 2 examples? I find them exactly the same.

Reply:
The one looks like mistake (used Resource but not `*** Settings ***` header) and the other looks like intentional comments section (some free text) also without header - `*** Comments ***`. It's to show that it can be easily matter of adding `*** Comments ***` or it's missing data header name (such as Settings, Test Cases etc)

Reply:
But it suggests that this is not properly tagged suite documentation, not a comment. Maybe it's better to show some deprecated test case or some data that instead of being commented out, can be put into `*** Comments ***` section?